### PR TITLE
Fixed links after docu modularization

### PR DESCRIPTION
**Pull request by krealyt**

### CODE REVIEW

File: docs/dev_tools/edm.rst [hunk 1]
```diff
@@ -75,7 +75,7 @@ refer to the `CPM Documentation <https://github.com/cpm-cmake/CPM.cmake/blob/mas
 
 Building everest
 ****************
-Make sure you have installed `ev_cli <doc_tutorial_EV-CLI.html>`_ first.
+Make sure you have installed `ev_cli <ev-cli.html>`_ first.
```
Comment (andistorm) [inline docs/dev_tools/edm.rst]:
```suggestion
Make sure you have installed `ev_cli <ev_cli.html>`_ first.
```

Comment (andistorm) [reply docs/dev_tools/edm.rst]: Wrong Link here

**Review by andistorm — CHANGES_REQUESTED** [review]
Looks good in gernal.
Maybe we should consider to rework all links and switch to Explicit or Implicit targets:
https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html

File: docs/dev_tools/edm.rst (hunk 1 above)
Comment (krealyt) [reply docs/dev_tools/edm.rst]: @andistorm OK, understood. Suggestion as I cannot do that currently: Let's fix the links in this PR now and do the "proper" repair as suggested by you in a second step. OK?

Comment (andistorm) [reply docs/dev_tools/edm.rst]: @krealyt, yes I made a commit to fix at least this single link (dash to underscore), If it ok for you, you can merge

**Review by andistorm — APPROVED** [review]

**Pull request merged**